src/coreclr/jit/jit.h

```diff
@@ -292,7 +292,8 @@
 // Arm64 Windows supports FEATURE_ARG_SPLIT, note this is different from
 // the official Arm64 ABI.
 // Case: splitting 16 byte struct between x7 and stack
-#if defined(TARGET_ARM) || defined(TARGET_ARM64)
+// LoongArch64's ABI supports FEATURE_ARG_SPLIT which splitting 16 byte struct between a7 and stack.
+#if (defined(TARGET_ARM) || (defined(TARGET_WINDOWS) && defined(TARGET_ARM64)) || defined(TARGET_LOONGARCH64))
```

Comment:
```suggestion
#if defined(TARGET_ARM) || defined(TARGET_ARM64) || defined(TARGET_LOONGARCH64)
```
`TARGET_WINDOWS` doesn't really exist in the Jit anymore.

What was the reason for its addition?

Reply:
Thanks very much. 
I will revert this and add only the `TARGET_LOONGARCH64`.